src/api/movie.tsx

```diff
@@ -5,28 +5,29 @@ const API_KEY = '49101d62654e71a2931722642ac07e5e'
 const yts = 'https://yts.am/api/v2/'
 
 export default {
-  movies(url) {
+  movies(url: string) {
```

Comment:
I suggest to rename `movie.tsx` to `movie.ts` since there is no JSX here.

Reply:
Done ✅